packages/device_info_plus/device_info_plus/macos/Classes/SystemUUID.swift

```diff
@@ -4,7 +4,13 @@ public struct SystemUUID {
     
     public static func getSystemUUID() -> String? {
         let dev = IOServiceMatching("IOPlatformExpertDevice")
-        let platformExpert: io_service_t = IOServiceGetMatchingService(kIOMasterPortDefault, dev)
+        var platformExpert: io_service_t
+
+        if #available(macOS 12, *) {
+            platformExpert = IOServiceGetMatchingService(kIOMainPortDefault, dev)
+        } else {
+            platformExpert = IOServiceGetMatchingService(kIOMasterPortDefault, dev)
+        }
```

Comment:
Thanks for your contribution 💪🏽

Nit: Could you change the blank lines, so the section regarding platformExpert looks more belonging together?

Reply:
> Thanks for your contribution 💪🏽
> 
> Nit: Could you change the blank lines, so the section regarding platformExpert looks more belonging together?

Thanks. I would like to do what you want but unfortunately can't right now, I will do it yesterday

Reply:
Updated this part myself. Thanks for contribution again.

Reply:
> Updated this part myself. Thanks for contribution again.

My apologies. The lack of time prevented me from doing this.